src/tags/if_block.rs

```diff
@@ -25,6 +26,25 @@ struct Conditional {
     if_false: Option<Template>,
 }
 
+fn contains_check(a: &Value, b: &Value) -> Result<bool> {
+    let b = b.as_str().ok_or("Right-hand side of contains operator must be a string")?;
+
+    match *a {
+        Value::Str(ref val) => Ok(val.contains(b)),
+        Value::Object(ref obj) => Ok(obj.contains_key(b)),
+        Value::Array(ref arr) => {
```

Comment:
I looked up `contains` but I thought it said it works for substrings and key checks.

Looking it up, instead its documented to work for substrings and array membership.
https://shopify.github.io/liquid/basics/operators/

But the implementation does work for objects's
- https://github.com/Shopify/liquid/blob/master/lib/liquid/condition.rb#L18
- https://ruby-doc.org/core-2.4.2/Hash.html#method-i-include-3F

So the only thing we're missing, compared to official liquid, is converting the right side to a string if the left side is a string.

That I'm perfectly happy postponing.  I've started a major revamp for `Value` that will make it easier to implement all of these random liquid conversions / coercions that take place.